modules/RTC/LocalSdpMunger.js

```diff
@@ -161,6 +161,44 @@ export default class LocalSdpMunger {
         return modified;
     }
 
+    /**
+     * Modifies 'cname', 'msid', 'label' and 'mslabel' by appending
+     * the id of {@link LocalSdpMunger#tpc} at the end, preceding by a dash
+     * sign.
+     *
+     * @param {MLineWrap} mediaSection - The media part (audio or video) of the
+     * session description which will be modified in place.
+     * @returns {void}
+     * @private
+     */
+    _transformMediaIdentifiers(mediaSection) {
+        const pcId = this.tpc.id;
+
+        for (const ssrcLine of mediaSection.ssrcs) {
+            switch (ssrcLine.attribute) {
+            case 'cname':
+            case 'label':
+            case 'mslabel':
+                ssrcLine.value = ssrcLine.value && `${ssrcLine.value}-${pcId}`;
+                break;
+            case 'msid': {
+                if (ssrcLine.value) {
+                    const streamAndTrackIDs = ssrcLine.value.split(' ');
+
+                    if (streamAndTrackIDs.length === 2) {
```

Comment:
do we not need to worry about the `else` case here?

Reply:
I was thinking about logging a warning, WDYT ?

Reply:
I didn't add it in the first place, because it's not like the first entry point for the local SDP where it would be supposed to validate SDP's correctness. I just thought that if we'd be adding checks like that everywhere maybe it would be too much... But I can add it

Reply:
i think a warning which prints out the msid would be good, yeah.

Reply:
done